llama-index-core/llama_index/core/workflow/__init__.py

```diff
@@ -0,0 +1,25 @@
+from llama_index.core.workflow.decorators import step
```

Comment:
General question, mostly unrelated to the PR. Would it be possible to use relative imports within a package? I never relative-import from parents (`..`) but I find it very handy to shorten imports of sibling and sub modules

Reply:
Hmm, I think throughout the repo we've used absolute imports for everything, so now its ingrained in me 😆